_episodes/07-raster-calculations.md

```diff
@@ -3,303 +3,367 @@ title: "Raster Calculations in Python"
 objectives:
-- "Perform a subtraction between two rasters using Python's built-in math operators to generate a Canopy Height Model (CHM)."
-- "Reclassify a continuous raster to a categorical raster using the CHM values."
+- "Carry out operations with two rasters using Python's built-in math operators."
+- "Reclassify a continuous raster to a categorical raster."
 keypoints:
 - "Python's built-in math operators are fast and simple options for raster math."
 - "numpy.digitize can be used to classify raster values in order to generate a less complicated map."
-- "DataArrays can be created from scratch from numpy arrays as well as read in from existing files."
+math: true
 ---
 
 > ## Things You’ll Need To Complete This Episode
 > See the [lesson homepage]({{ site.baseurl }}) for detailed information about the software,
 > data, and other prerequisites you will need to work through the examples in this episode.
+> TODO mention alternatives for data access
 {: .prereq}
 
-We often want to combine values of and perform calculations on rasters to create
-a new output raster. This episode covers how to subtract one raster from
-another using basic raster math. It also covers how to extract pixel values from a 
-set of locations - for example, a buffer region around locations at a field site.
+## Introduction
 
-## Raster Calculations in Python & Canopy Height Models
-We often want to perform calculations on two or more rasters to create a new
-output raster. For example, suppose we are interested in mapping the heights of trees across an entire field site. In that case, we might want to calculate the difference between the Digital Surface Model (DSM, tops of trees) and the Digital Terrain Model (DTM, ground level). The resulting dataset is referred to
-as a Canopy Height Model (CHM) and represents the actual height of trees,
-buildings, etc. with the influence of ground elevation removed.
+We often want to combine values of and perform calculations on rasters to create a new output raster. This episode
+covers how to perform basic math operations using raster datasets. It also illustrates how to match rasters with
+different resolutions so that they can be used in the same calculation. As an example, we will calculate a vegetation
+Difference Vegetation Index (NDVI) over one of the satellite scenes
 
-![Source: National Ecological Observatory Network (NEON)](../images/dc-spatial-raster/lidarTree-height.png) 
+### Normalized Difference Vegetation Index (NDVI)
+Suppose we are interested in monitoring vegetation fluctuations using satellite remote sensors. Scientists have defined
+a vegetation index to quantify the amount of green leaf vegetation using the light reflected in different wavelengths.
+This index, named Normalized Difference Vegetation Index (NDVI), exploits the fact that healthy green leaves strongly
+absorb red visible light while they mostly reflect light in the near infrared (NIR). The NDVI is computed as:
```

Comment:
Style based comment: The acronym NDVI is repeatedly defined within the first few paragraphs - could be reduced to just once.

Reply:
Fixed by dropping the first definition